### PR TITLE
Accurate error in removing a non-direct dependency.

### PR DESCRIPTION
**Pull request by laosb**
#6264 

The old error may confuse some beginners, and this PR should breaks nothing.
Also want to see the change in 1.3 :)

### CODE REVIEW

File: tools/cli/commands-packages.js [hunk 1]
```diff
@@ -2120,7 +2120,7 @@ main.registerCommand({
     } else if (! projectContext.projectConstraintsFile.getConstraint(packageName)) {
       // Check that we are using the package. We don't check if the package
       // exists. You should be able to remove non-existent packages.
-      Console.error(packageName  + " is not in this project.");
```
Comment (ramirez530) [inline tools/cli/commands-packages.js]: Delete this package

Comment (ramirez530) [inline tools/cli/commands-packages.js]: Delete this package

Comment (ramirez530) [inline tools/cli/commands-packages.js]: Remove

File: tools/cli/commands-packages.js [hunk 2]
```diff
@@ -2120,7 +2120,7 @@ main.registerCommand({
     } else if (! projectContext.projectConstraintsFile.getConstraint(packageName)) {
```
Comment (ramirez530) [inline tools/cli/commands-packages.js]: Use this package

File: tools/cli/commands-packages.js (hunk 1 above)
Comment (laosb) [inline tools/cli/commands-packages.js]: @ramirez530 I'm sorry that I didn't get what are you talking about. Could you make it clear?

**Comment by benjamn** [comment]
Looks good to me! Thanks for the PR.

**Comment by benjamn** [comment]
This will get released officially with Meteor 1.3.

**Pull request merged**